.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "image": "ghcr.io/citusdata/citus-devcontainer:dev-devcontainer",
+    "runArgs": [
+        "--cap-add=SYS_PTRACE",
+        "--security-opt",
+        "seccomp=unconfined",
```

Comment:
Need to double check this is the minimal viable set of flags, to also be able to attach the debugger